velox/exec/Spill.h

```diff
@@ -429,26 +429,27 @@ class FileSpillBatchStream : public BatchStream {
 /// start bit offset is used to calculate the partition number of spill data. It
 /// is required for the recursive spilling handling as we advance the start bit
 /// offset when we go to the next level of recursive spilling.
-struct SpillPartitionId {
+///
+/// NOTE: multiple of shards created from the same SpillPartition by split()
+/// will share the same id.
+class SpillPartitionId {
+ public:
   SpillPartitionId()
```

Comment:
Why do we need this constructor? Where do we use IDs created via this path?

Reply:
Removed.